onnxruntime/test/providers/provider_test_utils.cc

```diff
@@ -49,6 +49,10 @@ void Check<float>(const OpTester::Data& expected_data, const Tensor& output_tens
   threshold = 0.005f;
 #endif
 
+#ifdef USE_OPENVINO
+  threshold = 0.75f;
```

Comment:
while it's better than 1,  0.75 still seems kind of high (especially for fp32 computation)
Why do we require such a high threshold? i am concerned that we mask accuracy issues this way.

Reply:
what happens at the default 0.001f threshold? What tests are failing?

Reply:
removed this setting as it is no longer required to pass tests.